monitors/command.go

```diff
@@ -249,19 +249,19 @@ func isSameMonitor(a mackerel.Monitor, b mackerel.Monitor, flagNameUniquenessEna
 	return "", false
 }
 
-func validateRuleAnomalyDetectionScopes(v reflect.Value, label string) (bool, error) {
+func validateRuleAnomalyDetectionScopes(v reflect.Value, label string) error {
```

Comment:
A boolean in returned values is not needed because we can get the validation result only by error checking.